src/pages/Dashboard/Dashboard.tsx

```diff
@@ -70,6 +76,10 @@ export const Dashboard = memo(() => {
   const appIsMobile = isMobile || !isLargerThanMd
   const history = useHistory()
 
+  useEffect(() => {
+    if (!isConnected) walletDispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
+  }, [isConnected, walletDispatch])
+
```

Comment:
Wallet is the only one following a different pattern here, inherently requires a wallet, so hidden behind connect modal